examples/google-cloud-speech/output/google-cloud-speech/google-cloud-speech.gemspec

```diff
@@ -0,0 +1,30 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+Gem::Specification.new do |gem|
+  gem.name          = "google-cloud-speech"
+  gem.version       = "0.1.0"
+
+  gem.authors       = ["Google LLC"]
+  gem.email         = "googleapis-packages@google.com"
+  gem.description   = "google-cloud-speech is the official library for Cloud Speech API."
+  gem.summary       = "API Client library for Cloud Speech API"
+  gem.homepage      = "https://github.com/googleapis/googleapis"
```

Comment:
I think this is an unhelpful homepage. Even https://github.com/googleapis/google-cloud-ruby would be better.

Reply:
And https://github.com/googleapis/google-cloud-ruby/google-cloud-speech would be even better, IMO. But this might be something that should be handled by configuration instead of convention.